workflow-run/wfrun.ttl

```diff
@@ -84,6 +84,7 @@ wfrun:md5 rdf:type rdf:Property ;
 
 wfrun:registry rdf:type rdf:Property ;
                s:domainIncludes wfrun:ContainerImage ;
+               s:rangeIncludes s:Text ;
```

Comment:
shouldn't the range include a URL too? E.g., https://hub.docker.com/r/stain/jena/ instead of `stain/jena`